atomic_reactor/inner.py

```diff
@@ -413,33 +414,33 @@ class DockerBuildWorkflow(object):
     6. push it to registries
     """
 
-    def __init__(self, source=None, prebuild_plugins=None, prepublish_plugins=None,
-                 postbuild_plugins=None, exit_plugins=None, plugin_files=None,
-                 openshift_build_selflink=None, client_version=None,
-                 buildstep_plugins=None, **kwargs):
-        """
-        :param source: dict, where/how to get source code to put in image
-        :param prebuild_plugins: list of dicts, arguments for pre-build plugins
-        :param prepublish_plugins: list of dicts, arguments for test-build plugins
-        :param postbuild_plugins: list of dicts, arguments for post-build plugins
-        :param exit_plugins: list of dicts, arguments for exit plugins
-        :param plugin_files: list of str, load plugins also from these files
-        :param openshift_build_selflink: str, link to openshift build (if we're actually running
+    # The only reason this is here is to have something that unit tests can monkeypatch
+    _default_user_params = {}
+
+    def __init__(
+        self,
+        source: Source = None,
+        plugins: PluginsDef = None,
+        user_params: dict = None,
+        reactor_config_path: str = REACTOR_CONFIG_FULL_PATH,
+        plugin_files: List[str] = None,
```

Comment:
remove

Reply:
yeah I think it is time to get rid of it with new workflow params

Reply:
if this is the only thing, please do it in another PR, so this one can be merged